### PR TITLE
SPOI-9007 #resolve #comment Kryo Exception while re-deploying the Dim…

### PR DESCRIPTION
**Pull request by brightchen**
…ensionsComputationFlexibleSingleSchemaPOJO operator

### CODE REVIEW

**Comment by brightchen** [comment]
@tushargosavi @chaithu14 Could you please review

File: contrib/src/test/java/com/datatorrent/contrib/dimensions/DimensionComputationFaultTolerantTest.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    };
+
+    lma.prepareDAG(app, conf);
+
+    tupleCount = 0;
+    // Create local cluster
+    final LocalMode.Controller lc = lma.getController();
+    lc.runAsync();
+
+    while (tupleCount < tupleSize) {
```
Comment (tweise) [inline contrib/src/test/java/com/datatorrent/contrib/dimensions/DimensionComputationFaultTolerantTest.java]: timeout?

File: contrib/src/test/java/com/datatorrent/contrib/dimensions/DimensionComputationFaultTolerantTest.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
+package com.datatorrent.contrib.dimensions;
```
Comment (tushargosavi) [inline contrib/src/test/java/com/datatorrent/contrib/dimensions/DimensionComputationFaultTolerantTest.java]: license header ?

File: contrib/src/test/java/com/datatorrent/contrib/dimensions/FaultTolerantTestApp.java [hunk 3]
```diff
@@ -0,0 +1,171 @@
+package com.datatorrent.contrib.dimensions;
```
Comment (chaithu14) [inline contrib/src/test/java/com/datatorrent/contrib/dimensions/FaultTolerantTestApp.java]: License Header is missed.

**Comment by brightchen** [comment]
@tushargosavi @chaithu14 please review

**Comment by chaithu14** [comment]
Looks fine to me.

**Pull request merged**